### ISSUE TITLE
Please add type declarations to npm package

### ISSUE THREAD

**Issue by Stefaans** [christianp/asciimath2tex#17]
To use asciimath2tex in a TypeScript project, we need type definitions. The following works for me and may be a solution for you too:

In packages.json, specify the type definitions file:
```
"typings": "type.d.ts",
```

Add file type.d.ts:
```
export as namespace asciimath2tex;

class AsciiMathParser {
  constructor();
  parse(str): string;
}

export default AsciiMathParser;
```

With the type definitions in place, I can do:
```
import AsciiMathParser from 'asciimath2tex';
...
...
const parser = new AsciiMathParser();
const tex = parser.parse("int_(i=1)^10 x^2/2 dx");
```

**Comment by christianp** [comment]
Can you make a pull request that does this, please? I don't use TypeScript so don't know what needs to be done or how to test this.

**Comment by Stefaans** [comment]
Thank you Christian. I have submitted a pull request.

The solution was simpler than I expected. It was a simple case of specifying the types definition file in packages.json, and microbundle took care of the rest. The asciimath2tex.d.ts file that microbundle generates is bit more verbose than necessary, but good enough because it contains the essential definitions for the AsciiMathParser class and its parser method.

I was not familiar with microbundle until today. Learning something new every day :)

Stephen

**Comment by christianp** [comment]
fixed by #19

**Comment by Stefaans** [comment]
Christian, could you please bump the version number of the npm package? That way the new packages.json will be installed and the build performed with the types definition file. Working on the npm package is above my pay grade :)
